pkg/config/config.go

```diff
@@ -83,7 +86,10 @@ func (c *Config) SetupDictionary() error {
 		return err
 	}
 
-	if err := c.API.CreateDictionaryItem(c.Fastly.DictionaryName, "log_all_errors", "0"); err != nil {
+	if len(strings.TrimSpace(c.Fastly.LogAllErrors)) == 0 {
```

Comment:
Thinking about this more, i think we should construct the default config variable in line 46 with our defaults.  TOML allows you to to create a default 'config' and then anything it reads will override it.  I think we should do this for all the non-snippet variables in the config so that we can easily add backwards compatible defaults if the param is missing.

See here for example - https://play.golang.org/p/DJ76rVVE5a